test/jdk/java/net/ipv6tests/TcpTest.java

```diff
@@ -62,12 +65,10 @@ public class TcpTest extends Tests {
     public static void main (String[] args) throws Exception {
         checkDebug(args);
         if (ia4addr == null) {
-            System.out.println ("No IPV4 addresses: exiting test");
-            return;
+            throw new SkippedException("No IPV4 addresses: exiting test");
```

Comment:
Hello @sendaoYan, isn't an `import jtreg.SkippedException;` required for this change?

Reply:
This change require `import jtreg.SkippedException`, it's already import at line 42.

Reply:
You are right - i missed that in my review.